index.d.ts

```diff
@@ -1,5 +1,14 @@
 import * as React from 'react'
 
+type SavedQuery = {
+  id: string
+  name: string
+  query: string
+  default?: boolean
+  subgraphId?: number
+  versionoId?: string
```

Comment:
typo: `versionoId` -> `versionId`

Reply:
Good catch! I actually don't use it anywhere, so I'll remove it